lib/Option/SanitizerOptions.cpp

```diff
@@ -258,6 +258,22 @@ OptionSet<SanitizerKind> swift::parseSanitizerRecoverArgValues(
   return sanitizerRecoverSet;
 }
 
+bool swift::parseSanitizerAddressUseODRIndicator(
```

Comment:
I'm not convinced that the additional function really adds much value (over just inlining the single condition).

Reply:
I first had the same thought as you because I tried to write the code inline in `lib/Frontend/CompilerInvocation.cpp`. It turns out that doesn't work  because the code needs`StringRef toStringRef(const SanitizerKind kind)` and that is only callable from `lib/Option/SanitizerOptions.cpp`.

That is why I added `swift::parseSanitizerAddressUseODRIndicator(...)`. While we could export `toStringRef(const SanitizerKind kind)` I don't really think that's necessary at this time.

Reply:
Reasonable enough.  I think that really deserves a comment (perhaps in the implementation).

Reply:
Sure. I'll add a comment.

Reply:
Done.